src/Model.cpp

```diff
@@ -1,67 +1,88 @@
 
 #include "Model.h"
 #include <iostream>
+#include <fstream>
+#include <sstream>
+#include <filesystem>
 
 
 Model::Model(const std::string& path) : db(nullptr), dbPath(path) {
   if (sqlite3_open(dbPath.c_str(), &db) != SQLITE_OK) {
     std::cerr << "Can't open database: " << sqlite3_errmsg(db) << std::endl;
   } else {
     // init on instantiation
+    std::cout << "Opened database successfully" << std::endl;
     createTable();
   }
 }
 
-
 Model::~Model() {
   if (db) {
     sqlite3_close(db);
   }
 }
 
-
-bool
-Model::createTable() {
-  std::string sql = R"(
+bool Model::createTable() {
+    std::string sqlModels = R"(
         CREATE TABLE IF NOT EXISTS models (
             id INTEGER PRIMARY KEY AUTOINCREMENT,
-            short_name TEXT NOT NULL UNIQUE,
```

Comment:
What's the motivation for this?  The short name ideally needs to be unique to avoid a slew of UX confusion possibilities.